src/calculateSimilarity.ts

```diff
@@ -0,0 +1,39 @@
+import { disassembleHangul } from './disassemble';
+
+export function calculateSimilarity(a: string, b: string): number {
```

Comment:
사용자는 targetText, inputText라는 parameter명도 볼 수 있기 때문에 좀 더 구체화된 이름이면 좋을 것 같아요

아래는 TypeScript에 정의된 String의 method들이에요
<img width="634" alt="image" src="https://github.com/toss/es-hangul/assets/61593290/8672a095-e953-4402-99a3-46bcbb698452">


```suggestion
export function calculateSimilarity(targetText: string, inputText: string): number {
```

Reply:
이 함수의 목적에 맞게 인자의 이름을 정하는것은 너무 좋은 것 같아요!
하지만, 이 함수의 인자는 단순히 두 문자열이라서 인자의 이름을 정하기가 어려웠는데요, `calculateSimilarity`라는 함수 이름만으로도 두개의 인자가 비교대상이 되는 인자라는 것을 알아차릴 수 있다고 생각했어요. 

제안해주신 코드인 targetText, inputText 를 고려해봤을때, 제 생각은 다음과 같습니다!
1. 사용자는 targetText를 두번째 인자로 넣어도 함수의 결과값은 똑같고
2. 사용자의 input이 아닌, 이미 존재하는 두 값을 비교하려고 해도 이 함수를 사용할 수 있는데 input이라는 맥락으로 좁혀져도 괜찮은가?

**파라미터에 넣어주는 값의 순서가 바뀌어도 값은 동일하고, 함수명의로 인자의 역할이 충분히 설명 가능하다고 생각합니다**

더 떠오르는 구체적인 인자는 요런게 있을 것 같네용

```tsx
export function calculateSimilarity(comparison1: string, comparison2: string): number {
```

Reply:
보통은 1, 2와 같은 숫자를 넣지는 않아서 target, input과 같은 이름을 붙여주었는데 저도 고민되더라구요. 넵넵 일단 이 부분은 다음 버전에서 고려되어도 좋을 것 같기도 해요. 일단 배포하고 이슈따고 고민해봐도 좋을 것 같습니다

Reply:
이 함수를 일종의 이항연산자라고 생각하면 `left`, `right`라는 이름도 괜찮을 것 같은데 어떠신가요???

Reply:
left, right좋네요!